### ISSUE TITLE
Ansible install - utilize state present instead of latest

### ISSUE THREAD

**Issue by msheiny** [freedomofpress/securedrop#2727]
# Feature request

## Description

Using `state: latest` with the `apt` module forces apt_cache to refresh and is unnecessary on a re-run of the ansible install playbooks. The nightly cron job is responsible for this, lets try to speed up the ansible install runner so it's not so burdensome to run again over tor to make changes. 

Particuarly, we should swap all those cases of `latest` to utilize `present` instead.

This issue should probably be done in the same PR as #2724 . 

## User Stories

As a securedrop admin, when I want to make app changes and run `securedrop-admin install` again, I want that process to be as quick as possible.

**Comment by dachary** [comment]
so the way forward would be to check all apt and replace `latest` with `present` when we don't care about `latest`, right ? In some cases we actually want `latest` and we'll keep it.

**Comment by kushaldas** [comment]
The latest packages will be installed by the cron job every night. That will save the time for `apt` cache check for every `apt` install using the `latest` value.

**Comment by dachary** [comment]
Yes, but if package-1.0 is present and package-1.1 is available (and maybe we want the `latest`), if we're using `present` we will be stuck with package-1.0. Or am I missing something ?

**Comment by conorsch** [comment]
This would still be a solid improvement. We're needlessly re-updating the apt lists several times throughout the install flow. More recent versions of ansible-lint discourage use of state=latest. Would certainly review a PR if someone wanted to tackle this conversion.
